.travis.yml

```diff
@@ -105,16 +105,18 @@ matrix:
                   packages:
                       - golang-1.10
           compiler: gcc
-          env:  EXTENDED_TEST="yes" CONFIG_OPTS="--debug enable-ssl3 enable-ssl3-method enable-weak-ssl-ciphers enable-external-tests enable-buildtest-c++" BORINGSSL_TESTS="yes" CXX="g++" TESTS=95
+          # External tests pyca-cryptography and krb5 temporarily disabled due to long term travis failures
```

Comment:
should this comment be updated given that krb5 tests are not disabled?

Reply:
Amended my last fixup to adjust the comment and add `[extended tests]`, see e643b8089588097504dcb7ddf5382affb22504bd.